src/main/java/uk/gov/hmcts/reform/prl/services/ManageOrderEmailService.java

```diff
@@ -174,7 +238,21 @@ private List<PartyDetails> getRespondents(CaseData caseData) {
             .collect(Collectors.toList());
     }
 
-
+    private List<Map<String, List<String>>> getRespondentSolicitor(CaseDetails caseDetails) {
+        CaseData caseData = emailService.getCaseData(caseDetails);
+        return caseData
+            .getRespondents()
```

Comment:
DA case check here?

Reply:
The requirement to send emails to the Respondent solicitor was only for CA applications, so DA applications are not catered to here. please refer PRL-1974